pkg/granted/credentials.go

```diff
@@ -503,3 +508,76 @@ var ExportCredentialsCommand = cli.Command{
 		return nil
 	},
 }
+
+var RotateCredentialsCommand = cli.Command{
+	Name:      "rotate",
+	Usage:     "Generates new access key for the profile in AWS, and updates the profile",
+	ArgsUsage: "[<profile>]",
+	Action: func(c *cli.Context) error {
+		profileName := c.Args().First()
```

Comment:
You can make `profile` as optional flag like in your TokenExpiryCommand PR which will give a nice Usage description to users. Currently, it's not intuitive if we can bypass the profile selection if profile name is passed as first argument.